### PR TITLE
chore: update

### PR DESCRIPTION
**Pull request by alexander-akait**
BREAKING CHANGE: minimum supported Node.js version is `10.13`, use `md4` by default for hashes

This PR contains a:

- [ ] **bugfix**
- [ ] new **feature**
- [x] **code refactor**
- [ ] **test update** 
- [ ] **typo fix**
- [x] **metadata update**

### Motivation / Use-Case

The next version of `cache-loader`

### Breaking Changes

Yes

### Additional Info

No

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,46 +34,44 @@
...
     "buffer-json": "^2.0.0",
-    "find-cache-dir": "^3.0.0",
-    "loader-utils": "^1.2.3",
-    "mkdirp": "^0.5.1",
+    "find-cache-dir": "^3.3.1",
+    "loader-utils": "^2.0.0",
+    "mkdirp": "^1.0.4",
     "neo-async": "^2.6.1",
-    "schema-utils": "^2.0.0"
+    "schema-utils": "^2.6.6"
```
Comment (matamatanot) [inline package.json:45]:
```suggestion
    "schema-utils": "^3.0.0"
```

**Review by matamatanot — CHANGES_REQUESTED** [review]
What about `schema-utils` 3.0.0?

```
import { validate } from 'schema-utils';

validate(schema, options, {
    name: 'Cache Loader',
    baseDataPath: 'options',
});
```

**Comment by evilebottnawi** [comment]
No need, please remove `cache-loader` and migrate on webpack@5 (look at the `cache` option https://webpack.js.org/configuration/other-options/#cache), this loader will be deprecated in the near future

**Comment by matamatanot** [comment]
@evilebottnawi 
Thanks!

**Pull request closed without merging**